src/clusterfuzz/_internal/issue_management/google_issue_tracker/issue_tracker.py

```diff
@@ -394,7 +394,13 @@ def save(self, new_comment=None, notify=True):
       if priority:
         self._data['issueState']['priority'] = priority
       if os:
-        self._data['issueState'][_CHROMIUM_OS_CUSTOM_FIELD_ID] = os
+        # TODO(rmistry): _update_issue will need to handle this as well.
```

Comment:
@oliverchang is this correct? Could the platform change via _update_issue ?

Reply:
I think it can change in some circumstances, so yes.